paasta_tools/utils.py

```diff
@@ -818,14 +818,21 @@ def list_clusters(service=None, soa_dir=DEFAULT_SOA_DIR):
         service = '*'
     srv_path = os.path.join(soa_dir, service)
 
+    if instance_type == 'marathon' or instance_type == 'chronos':
+        instance_types = instance_type
+    else:
+        instance_types = 'marathon|chronos'
+
+    search_re = r'/.*/(' + instance_types + r')-([0-9a-z-]*).yaml$'
+
     for yaml_file in glob.glob('%s/*.yaml' % srv_path):
-        cluster_re_match = re.search('/.*/(marathon|chronos)-([0-9a-z-]*).yaml$', yaml_file)
+        cluster_re_match = re.search(search_re, yaml_file)
         if cluster_re_match is not None:
             clusters.add(cluster_re_match.group(2))
     return sorted(clusters)
 
 
-def list_all_instances_for_service(service, cluster=None, instance_type=None, soa_dir=DEFAULT_SOA_DIR):
+def list_all_instances_for_service(service, instance_type=None, soa_dir=DEFAULT_SOA_DIR):
```

Comment:
ah ok, this makes sense. I see why you're changing this in ``cli/utils.py`` now.